src/Microsoft.Restier.Core/ApiContextExtensions.cs

```diff
@@ -77,7 +201,9 @@ public static class ApiContextExtensions
 
             try
             {
-                var model = await builder.GetModelAsync(new InvocationContext(context), cancellationToken);
+                var buildContext = new InvocationContext(context);
+                var model = await builder.GetModelAsync(buildContext, cancellationToken);
+                buildContext.DisposeScope();
```

Comment:
+1. But then I guess we also need to make `ApiContext` disposable to keep consistent? Anyway we can do this in a separate task.